homeassistant/components/netgear_lte/__init__.py

```diff
@@ -35,16 +37,21 @@
 
 SENSOR_SCHEMA = vol.Schema({
     vol.Required(CONF_MONITORED_CONDITIONS):
```

Comment:
Make this optional and put the default sensor types here.

Reply:
I tried that and it didn't work. I think `ensure_list` turns a default `None` into an empty list and my default gets ignored. I could be wrong, was in a bit of a hurry to leave.

The rest of your comments follow from this.

Reply:
Where's the default `None` coming from? We shouldn't pass a default `None`?

Reply:
OK, let me check again tonight.

Reply:
You are right. Trying to reproduce my issue, I think I was testing with
```yaml
netgear_lte:
  host: 127.0.0.1
  password: homeassistant
  sensor:
```
... so it was `sensor` that was `None` but that is not a problem.